### ISSUE TITLE
spaceline-spacemacs-theme issue

### ISSUE THREAD

**Issue by brabalan** [bbatsov/zenburn-emacs#214]
I use spaceline (https://github.com/TheBB/spaceline), and I see something slightly off:

<img width="45" alt="screen shot 2015-11-26 at 15 42 53" src="https://cloud.githubusercontent.com/assets/1021637/11425344/93a412e0-9454-11e5-9ff1-a60c5962582b.png">

It seems the very top line of the minibuffer uses the size from the bottom. Here is my spaceline configuration:
``` emacs-lisp
(require 'spaceline-config)
(spaceline-spacemacs-theme)
(setq powerline-default-separator 'rounded)
(setq spaceline-highlight-face-func 'spaceline-highlight-face-evil-state)
```

**Comment by thegeorgeous** [comment]
Are you using OS X?

**Comment by brabalan** [comment]
Yes. But I've since moved to spacemacs, and I do not see any problem there.
